### PR TITLE
Add MedicationRequest Search and Read Documentation

### PR DESCRIPTION
**Pull request by beichhor**
Screen Shot 2019-12-11 at 3 10 41 PM
Screen Shot 2019-12-11 at 3 11 08 PM

### CODE REVIEW

File: content/millennium/r4/medications/medication_request.md [hunk 1]
```diff
@@ -0,0 +1,144 @@
...
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
+ `patient`                | This, or `_id`     | [`reference`] | The specific patient to return MedicationRequests for. Example: `12345`
+ `status`                 | N                  | [`token`]     | The status of the medication, may be a list separated by commas. Example: `active,completed`
+ `-timing-boundsPeriod`   | N                  | [`token`]     | The date-time which should fall within the dosageInstruction.timing.repeat.boundsPeriod the medication should be given to the patient. Must be prefixed by ‘ge’. Example: `ge2014-05-19T20:54:02.000Z`
+ `_lastUpdated`           | N                  | [`date`]      | An explicit or implied date-time range within which the most recent clinically relevant update was made to the medication. Must be prefixed by ‘ge’ or ‘le’. Example: `ge2014-05-19T20:54:02.000Z`
+ `_count`                 | N                  | [`number`]    | The maximum number of results to include in a page. Example: `50`
+
+ * The `_lastUpdated` and `-timing-boundsPeriod` parameter may be provided:
```
Comment (beichhor) [inline content/millennium/r4/medications/medication_request.md]: Need to clarify on this.

File: content/millennium/r4/medications/medication_request.md [hunk 2]
```diff
@@ -0,0 +1,144 @@
...
+
+### Parameters
+
+ Name      | Required?          | Type          | Description
+-----------|--------------------|---------------|-----------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
+ `patient`                | This, or `_id`     | [`reference`] | The specific patient to return MedicationRequests for. Example: `12345`
+ `status`                 | N                  | [`token`]     | The status of the medication, may be a list separated by commas. Example: `active,completed`
+ `-timing-boundsPeriod`   | N                  | [`token`]     | The date-time which should fall within the dosageInstruction.timing.repeat.boundsPeriod the medication should be given to the patient. Must be prefixed by ‘ge’. Example: `ge2014-05-19T20:54:02.000Z`
+ `_lastUpdated`           | N                  | [`date`]      | An explicit or implied date-time range within which the most recent clinically relevant update was made to the medication. Must be prefixed by ‘ge’ or ‘le’. Example: `ge2014-05-19T20:54:02.000Z`
```
Comment (jfrhong) [inline content/millennium/r4/medications/medication_request.md]: should we mention that time must be provided?

File: content/millennium/r4/proprietary-codes-and-systems.md [hunk 3]
```diff
@@ -498,6 +593,14 @@ The Account Resource supports proprietary systems for:
 
 * Account.identifier.system is [Account Number](#account-number) for financial accounts.
 
+### Medications
+
+<h4>MedicationRequest</h4>
+
+The MedicationRequest Resource supports proprietary systems for:
+
+* MedicationRequest.category is [Account Number](#account-number) for financial accounts.
```
Comment (jfrhong) [inline content/millennium/r4/proprietary-codes-and-systems.md]: Need to update the description

File: content/millennium/r4/proprietary-codes-and-systems.md [hunk 4]
```diff
@@ -509,3 +612,14 @@ This system is the account number of a financial account.
       "system": "https://fhir.cerner.com/accountnumber",
       "value": "5646"
     }
+
+##### Medication Request Category
+
+This system is the category and is only for a pharmacy charge-only order. The code system currently only
+contains the `charged-only` code. The code system may be expanded at a future date.
```
Comment (jfrhong) [inline content/millennium/r4/proprietary-codes-and-systems.md]: "charged-only" => "charge-only"

File: content/millennium/r4/medications/medication_request.md [hunk 5]
```diff
@@ -0,0 +1,144 @@
...
+---
+
+# MedicationRequest
+
+* TOC
+{:toc}
+
+## Overview
+
+The Medication Request resource provides orders for all medications along with administration instructions for a
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/medication_request.md]: I would prefer to not insert manual line breaks

File: content/millennium/r4/medications/medication_request.md [hunk 6]
```diff
@@ -0,0 +1,144 @@
...
+patient in both the inpatient and outpatient setting (orders/prescriptions filled by a pharmacy and discharge
+medication orders). This resource also includes a patients historical or documented home medications reported
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/medication_request.md]: patient's

File: content/millennium/r4/medications/medication_request.md (hunk 5 above)
Comment (MaxPhilips) [inline content/millennium/r4/medications/medication_request.md]: it seems for non-med resources we do one word (e.g. AllergyIntolerance) but for meds we have separated them. I'd prefer to use the FHIR name so MedicationRequest throughout

File: content/millennium/r4/medications/medication_request.md [hunk 7]
```diff
@@ -0,0 +1,144 @@
...
+The following fields are returned if valued:
+
+* [MedicationRequest id]( https://hl7.org/fhir/r4/resource-definitions.html#Resource.id){:target="_blank"}
+* [Status](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.status){:target="_blank"}
+* [Status Reason](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.statusReason){:target="_blank"}
+* [Intent](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.intent){:target="_blank"}
+* [Category](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.category){:target="_blank"}
+* [Priority](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.priority){:target="_blank"}
+* [Reported Boolean](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.reported_x_){:target="_blank"}
+* [Medication](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.medication_x_){:target="_blank"}
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/medication_request.md]: if we write Reported Boolean I suppose we should write Medication Reference/CodeableConcept. or is it the case that we return both references and ccs?

File: content/millennium/r4/medications/medication_request.md [hunk 8]
```diff
@@ -0,0 +1,144 @@
...
+* [Dosage Instructions](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dosageInstruction){:target="_blank"}
+   * [Text](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.text){:target="_blank"}
+   * [Additional Instructions](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.additionalInstruction){:target="_blank"}
+   * [Patient Instructions](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.patientInstruction){:target="_blank"}
+   * [Timing Schedule](https://hl7.org/fhir/r4/datatypes-definitions.html#Timing){:target="_blank"}
+   * [As Needed](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.asNeeded_x_){:target="_blank"}
+   * [Body Site](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.site ){:target="_blank"}
+   * [Route](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.route){:target="_blank"}
+   * [Dose and Rate](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.doseAndRate){:target="_blank"}
+[Dispensing Details](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest){:target="_blank"}
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/medication_request.md]: need a bullet to start this line

File: content/millennium/r4/medications/medication_request.md [hunk 9]
```diff
@@ -0,0 +1,144 @@
...
+    * [Validity Period](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.validityPeriod ){:target="_blank"}
+    * [Number of Refills](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.numberOfRepeatsAllowed){:target="_blank"}
+    * [Dispense Quantity](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.quantity){:target="_blank"}
+    * [Expected Days Supply per Dispense](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.expectedSupplyDuration){:target="_blank"}
+    * [Intended Dispenser](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dispenseRequest.performer){:target="_blank"}
+* [Substitution Allowed Boolean](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.substitution.allowed_x_){:target="_blank"}
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/medication_request.md]: might want to break this down into multiple bulleted layers like other complex fields

File: content/millennium/r4/medications/medication_request.md [hunk 10]
```diff
@@ -0,0 +1,144 @@
...
+
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name      | Required?          | Type          | Description
+-----------|--------------------|---------------|-----------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource id associated with the resource.
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/medication_request.md]: could add an Example

File: content/millennium/r4/medications/medication_request.md [hunk 11]
```diff
@@ -0,0 +1,144 @@
...
+ `patient`                | This, or `_id`     | [`reference`] | The specific patient to return MedicationRequests for. Example: `12345`
+ `status`                 | N                  | [`token`]     | The status of the medication, may be a list separated by commas. Example: `active,completed`
+ `-timing-boundsPeriod`   | N                  | [`token`]     | The date-time which should fall within the dosageInstruction.timing.repeat.boundsPeriod the medication should be given to the patient. Must be prefixed by ‘ge’. Example: `ge2014-05-19T20:54:02.000Z`
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/medication_request.md]: let's put dosageInstruction... inside backticks

File: content/millennium/r4/medications/medication_request.md [hunk 12]
```diff
@@ -0,0 +1,144 @@
...
+ `_lastUpdated`           | N                  | [`date`]      | An explicit or implied date-time range within which the most recent clinically relevant update was made to the medication. Must be prefixed by ‘ge’ or ‘le’. Example: `ge2014-05-19T20:54:02.000Z`
+ `_count`                 | N                  | [`number`]    | The maximum number of results to include in a page. Example: `50`
+
+ * The `_lastUpdated` and `-timing-boundsPeriod` parameter may be provided:
+     * once with a prefix `ge` or `le` representing the earliest date or latest date. (e.g. `date=ge2015-01-01`, `date=le2016-01-01`)
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/medication_request.md]: if the prefixes here are backticked, backtick all prefixes. or if we want to single quote them, single quote all

File: content/millennium/r4/proprietary-codes-and-systems.md (hunk 3 above)
Comment (MaxPhilips) [inline content/millennium/r4/proprietary-codes-and-systems.md]: probably need to update account number

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 13]
```diff
@@ -0,0 +1,2128 @@
...
+                "reference": "Patient/1316024",
+                "display": "PETERS, TIMOTHY"
+            },
+            "encounter": {
+                "reference": "Encounter/6188499"
+            },
+            "authoredOn": "2016-06-24T10:38:59.000-05:00",
+            "requester": {
+                "reference": "Practitioner/1852008",
+                "display": "Hong, Jennifer"
```
Comment (MaxPhilips) [inline lib/resources/example_json/r4_examples_medication_request.rb]: doctor who

File: lib/resources/example_json/r4_examples_medication_request.rb [hunk 14]
```diff
@@ -0,0 +1,2128 @@
...
+                      "text": "Misc Prescription (Freetext)"
+                  },
+                  "subject": {
+                      "reference": "Patient/1316024",
+                      "display": "PETERS, TIMOTHY"
+                  },
+                  "encounter": {
+                      "reference": "Encounter/6188499"
+                  },
+
```
Comment (MaxPhilips) [inline lib/resources/example_json/r4_examples_medication_request.rb]: 25 is way too many example med requests

File: content/millennium/r4/medications/medication_request.md [hunk 15]
```diff
@@ -0,0 +1,144 @@
+---
```
Comment (averyallen) [inline content/millennium/r4/medications/medication_request.md]: Noticed this while using this an example of what needs to be added for DocumentReference.

We've been using hyphen over comma to separate words in file names, medication_request.md -> medication-request.md

File: content/millennium/r4/medications/medication_request.md (hunk 7 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: We can return either a medicationCodeableConcept or a medicationReference.

File: content/millennium/r4/medications/medication_request.md (hunk 9 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: We can, but this is the only field we will return under substitution.

File: content/millennium/r4/medications/medication_request.md (hunk 15 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: Resolved in: https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce#diff-91cabf44fbcf0c5aaa861d340e57d246R12

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 14 above)
Comment (beichhor) [reply lib/resources/example_json/r4_examples_medication_request.rb]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/61f407d4385bd72a2b599551739c8e1fd09b3d67

File: lib/resources/example_json/r4_examples_medication_request.rb (hunk 13 above)
Comment (beichhor) [reply lib/resources/example_json/r4_examples_medication_request.rb]: How about P&R references? https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce#diff-72f45ea15972eed5522e5c8e33200782R125

File: content/millennium/r4/proprietary-codes-and-systems.md (hunk 3 above)
Comment (beichhor) [reply content/millennium/r4/proprietary-codes-and-systems.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce?file-filters%5B%5D=.md#diff-6fdcb00b5b1b6edd51b687794bea6b95R602

File: content/millennium/r4/medications/medication_request.md (hunk 12 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/f8adf8960cba9c644ecdc5477491918f0f44486d#diff-91cabf44fbcf0c5aaa861d340e57d246R77

File: content/millennium/r4/medications/medication_request.md (hunk 11 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/f8adf8960cba9c644ecdc5477491918f0f44486d#diff-91cabf44fbcf0c5aaa861d340e57d246R76

File: content/millennium/r4/medications/medication_request.md (hunk 10 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/f8adf8960cba9c644ecdc5477491918f0f44486d#diff-91cabf44fbcf0c5aaa861d340e57d246R73

File: content/millennium/r4/medications/medication_request.md (hunk 8 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/f8adf8960cba9c644ecdc5477491918f0f44486d#diff-91cabf44fbcf0c5aaa861d340e57d246R42

File: content/millennium/r4/medications/medication_request.md (hunk 5 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce?file-filters%5B%5D=.md#diff-91cabf44fbcf0c5aaa861d340e57d246R12

File: content/millennium/r4/medications/medication_request.md (hunk 6 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce?file-filters%5B%5D=.md#diff-91cabf44fbcf0c5aaa861d340e57d246R14

File: content/millennium/r4/medications/medication_request.md (hunk 5 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/f8adf8960cba9c644ecdc5477491918f0f44486d#diff-91cabf44fbcf0c5aaa861d340e57d246L12

File: content/millennium/r4/proprietary-codes-and-systems.md (hunk 4 above)
Comment (beichhor) [reply content/millennium/r4/proprietary-codes-and-systems.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce?file-filters%5B%5D=.md#diff-6fdcb00b5b1b6edd51b687794bea6b95R618

File: content/millennium/r4/proprietary-codes-and-systems.md (hunk 3 above)
Comment (beichhor) [reply content/millennium/r4/proprietary-codes-and-systems.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce?file-filters%5B%5D=.md#diff-6fdcb00b5b1b6edd51b687794bea6b95R602

File: content/millennium/r4/medications/medication_request.md (hunk 2 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce?file-filters%5B%5D=.md#diff-91cabf44fbcf0c5aaa861d340e57d246R84

File: content/millennium/r4/medications/medication_request.md (hunk 1 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/909f357d1c858d588741c6b3265da1827c6a43ce?file-filters%5B%5D=.md#diff-91cabf44fbcf0c5aaa861d340e57d246R84

File: content/millennium/r4/medications/medication-request.md [hunk 16]
```diff
@@ -0,0 +1,137 @@
...
+
+<%= authorization_types(practitioner: true, patient: false, system: true) %>
+
+### Parameters
+
+ Name      | Required?          | Type          | Description
+-----------|--------------------|---------------|-----------------------------------------------------------------------------------------------------
+ `_id`                    | This, or `patient` | [`token`]     | The logical resource id associated with the resource. Example: `12345`
+ `patient`                | This, or `_id`     | [`reference`] | The specific patient to return MedicationRequests for. Example: `12345`
+ `status`                 | N                  | [`token`]     | The status of the medication, may be a list separated by commas. Example: `active,completed`
```
Comment (jfrhong) [inline content/millennium/r4/medications/medication-request.md]: should we say anything about the system being provided?

File: content/millennium/r4/medications/medication_request.md (hunk 9 above)
Comment (MaxPhilips) [reply content/millennium/r4/medications/medication_request.md]: yeah, balance between observing bullet rules about not having single lower level bullets and matching the indentation rules we set up elsewhere in the section. i'm ok with either way but my preference would be adding another layer, you can mark this resolved when finished

File: content/millennium/r4/proprietary-codes-and-systems.md [hunk 17]
```diff
@@ -498,6 +593,14 @@ The Account Resource supports proprietary systems for:
 
 * Account.identifier.system is [Account Number](#account-number) for financial accounts.
 
+### Medications
+
+<h4>MedicationRequest</h4>
+
+The MedicationRequest Resource supports proprietary systems for:
+
+* MedicationRequest.category is [Medication Request Category](#medication-request-category) for medication orders.
```
Comment (jfrhong) [inline content/millennium/r4/proprietary-codes-and-systems.md]: nit: for medication requests

File: content/millennium/r4/medications/medication_request.md (hunk 15 above)
Comment (MaxPhilips) [reply content/millennium/r4/medications/medication_request.md]: update toc links when renaming files

File: content/millennium/r4/medications/medication-request.md [hunk 18]
```diff
@@ -0,0 +1,137 @@
...
+* [Requester](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.requester){:target="_blank"}
+* [Reason Code](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.reasonCode){:target="_blank"}
+* [Course of Therapy Type](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.courseOfTherapyType){:target="_blank"}
+* [Note](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.note){:target="_blank"}
+* [Dosage Instructions](https://hl7.org/fhir/r4/medicationrequest-definitions.html#MedicationRequest.dosageInstruction){:target="_blank"}
+   * [Text](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.text){:target="_blank"}
+   * [Additional Instructions](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.additionalInstruction){:target="_blank"}
+   * [Patient Instructions](https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.patientInstruction){:target="_blank"}
+   * [Timing Schedule](https://hl7.org/fhir/r4/datatypes-definitions.html#Timing){:target="_blank"}
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/medication-request.md]: should this link to https://hl7.org/fhir/r4/dosage-definitions.html#Dosage.timing instead?

File: content/millennium/r4/medications/medication-request.md [hunk 19]
```diff
@@ -0,0 +1,137 @@
...
+
+### Example
+
+#### Request
+
+    GET https://fhir-open.sandboxcerner.com/r4/0b8a0111-e8e6-4c26-a91c-5069cbc6b1ca/MedicationRequest?patient=1316020
+
+#### Response
+
+<%= headers status: 200 %>
```
Comment (MaxPhilips) [inline content/millennium/r4/medications/medication-request.md]: ensure headers render with application/fhir+json

Comment (beichhor) [reply content/millennium/r4/medications/medication-request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/7bee2124b41c1252ae9347951199fbd8db627be3#diff-91cabf44fbcf0c5aaa861d340e57d246R86

File: content/millennium/r4/proprietary-codes-and-systems.md (hunk 17 above)
Comment (beichhor) [reply content/millennium/r4/proprietary-codes-and-systems.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/7bee2124b41c1252ae9347951199fbd8db627be3#diff-6fdcb00b5b1b6edd51b687794bea6b95R602

File: content/millennium/r4/medications/medication-request.md (hunk 16 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication-request.md]: Hopefully a link will suffice: https://github.com/cerner/fhir.cerner.com/pull/332/commits/7bee2124b41c1252ae9347951199fbd8db627be3#diff-91cabf44fbcf0c5aaa861d340e57d246R75

File: content/millennium/r4/medications/medication-request.md (hunk 18 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication-request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/7bee2124b41c1252ae9347951199fbd8db627be3#diff-91cabf44fbcf0c5aaa861d340e57d246R37

File: content/millennium/r4/medications/medication_request.md (hunk 15 above)
Comment (beichhor) [reply content/millennium/r4/medications/medication_request.md]: https://github.com/cerner/fhir.cerner.com/pull/332/commits/7bee2124b41c1252ae9347951199fbd8db627be3?file-filters%5B%5D=.html#diff-1abdee660a30300c05753561e598f2dcR64

**Comment by JFisk42** [comment]
Verified the published version at: https://fhir.cerner.com/millennium/r4/medications/medication-request/

Screenshot_2019-12-18 MedicationRequest R4 API

**Pull request merged**